### PR TITLE
Add idr-s3 docs

### PR DESCRIPTION
**Pull request by manics**
Add a short doc about the idr-ftp S3 Minio upload server

### CODE REVIEW

**Review by joshmoore — APPROVED** [review]
:+1:

File: docs/idr-s3.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# IDR S3 upload server
+
+The IDR S3 submission server is co-located with the IDR FTP server.
```
Comment (sbesson) [inline docs/idr-s3.md:3]: This made me wonder: we have no similar documentation describing the FTP server deployment?

**Review by sbesson — APPROVED** [review]
Thanks, overall makes sense. Eventually, I can also see this document cross-linked from the submission workflow as we will need to create Minio users per submission so it is good to have a starting page.

One general question re the layout of these docs. It looks like originally the documents were close the deployment procedure (`provisioning`, `deployment`). More recently, we introduced documents per service: `monitoring`, `s3`. Would a top-level index e.g. a table of contents for the various topics covered by these docs help the discoverability?

**Comment by manics** [comment]
I've renamed the doc and added some information on the FTP server, and also linked it from the README.

**Comment by sbesson** [comment]
Thanks.

**Pull request merged**